client/verta/verta/client.py

```diff
@@ -1735,3 +1736,34 @@ def find_datasets(
             datasets = datasets.find(predicates)
 
         return datasets
+
+    def _create_workspace(self, org_id, workspace_name, resource_action_groups):
+        """
+        creates a workspace with a custom role for all users.
+
+        Parameters
+        ----------
+        org_id : str
+            ID of organization in which workspace is created.
+        workspace_name : str
+            name of workspace.
+        resource_action_groups : [_Role.RoleResourceActions]
+            role description with allowed actions for resource types.
+
+        Returns
+        -------
+        :class:`~verta.tracking._workspace.Workspace`
+
+        """
+        org = OrganizationV2(self._conn, org_id)
+        groups = org.get_groups()
+        all_users_group_id = next(iter(set(group.id for group in groups if group.name == "All Users")))
+        admins_group_id = next(iter(set(group.id for group in groups if group.name == "Admins")))
+        super_user_role_id = next(iter(set(role.id for role in org.get_roles() if role.name == "Super User")))
```

Comment:
I'm a bit surprised we don't have a way to get group/role by name from UAC 😮 But this is...alright we me, if it needs to be done this way.

Reply:
yes, we don't have search by name at the initial version